### PR TITLE
test(global drafts-onboarding-tour-tip): New test for Global Drafts onboarding

### PR DESCRIPTION
**Pull request by jgilliam17**
#### Summary

Adding new test for Global Drafts onboarding tour tip
Jira ticket: https://mattermost.atlassian.net/browse/MM-49179
#### Is the feature in stable branch or under review?
- [] Feature in stable branch (master, main)?
- [] Feature still under review? If yes, list all related pull requests for reference.

#### Test client

- [] Browser
- [] Mobile App (version, OS)
- [] Desktop App (version, OS)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/55"></a>

### CODE REVIEW

**Comment by jgilliam17** [comment]
Originally I had step 2 incorporated into step 1, and then decided to split it out into a separate step.
Thoughts?
I'm ok with either approach.

File: data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+2. Add some text to the text box e.g. pineapple
+
+   1. Verify global Drafts item is added to LHS
+   2. Verify Global Drafts Onboarding Tour Point is visible with "Not Now" and "View Drafts" buttons
+   3. Verify text "With the new Drafts view, all of your unfinished messages are collected in one place. Return here to read, edit, or send draft messages."
+
+3. Click on View Drafts
+
+   1. Verify global draft inbox opens
+   2. Verify draft created in step 2 is present
```
Comment (saturninoabril) [inline data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip:72]: I'd prefer to merge this step 2 to 1 so some steps are not repeated as it's likely the same when doing manual and automated tests.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Jelena! LGTM, except for couple of request.

File: data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
```
[the remark this answers is not in the document]
Comment (jgilliam17) [reply data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip]: Oh, good catch. Done

File: data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip (hunk 1 above)
Comment (jgilliam17) [reply data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip]: Changed. Can you take another look? Thanks.

File: data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Open Town Square on Team1
+
+2. Add some text to the text box e.g. pineapple
+
+   1. Verify global Drafts item is added to LHS
+   2. Verify Global Drafts Onboarding Tour Point is visible with "Not Now" and "View Drafts" buttons
+   3. Verify text "With the new Drafts view, all of your unfinished messages are collected in one place. Return here to read, edit, or send draft messages."
+
+3. Click on Not Now button on the tour
```
Comment (mattermost-quality) [inline data/test-cases/channels/global-drafts/Global-Drafts-Onboarding-Tourtip.md:76]: nit: for consistency
```suggestion
3. Click on "Not Now" button on the tour
```

**Review by mattermost-quality — APPROVED** [review]
Thanks Jelena, looks good to me!

**Comment by saturninoabril** [comment]
(Oh no... it should not be the bot account. My bad).

**Review by saturninoabril — APPROVED** [review]
LGTM, thanks Jelena!

**Comment by saturninoabril** [comment]
Test created as [MM-T5378](https://github.com/mattermost/mattermost-test-management/blob/main/data/test-cases/channels/global-drafts/MM-T5378.md).

**Pull request merged**